### ISSUE TITLE
Combine relative and absolute range picker

### ISSUE THREAD

**Issue by yomete** [parca-dev/parca#4748]
Just to go back in time to understand why we redesigned the range picker in the first place, the original issue can be seen [here](https://github.com/parca-dev/parca/issues/4024).

`We've had user feedback saying that the two-column time picker is confusing as they read in the order of an academic paper, which is unexpected.`

One thing that came out from the discussion in the issue linked above (and was implemented) is that we wanted to not have the preset ranges as different buttons but instead be able to change them easily using the + and - buttons.

And now to today's problem, the feedback from users is the text below the range picker to switch between the relative and absolute isn't clear enough/easy to miss.

![Image](https://github.com/parca-dev/parca/assets/9016992/0a168cdc-b673-4aa9-b789-3ff5017404ba)

So we're fine with how the relative and absolute range picker works, we just want to improve how to switch between either of them.

If we want to do a combination of both range pickers (relative and absolute), I imagine we can implement it in such a way that:

1. We remove any action buttons (that might increase/decrease the range of the query) so that it simply just displays the range as a text e.g. `Last 30 mins` or `2024-06-11 11:28:23 to 2024-06-11 11:43:23`. 
2. Users can then click on that range text to open up a modal which then shows both the relative and absolute range pickers as they are right now.
3. They can then decide to apply a range using either the relative or absolute picker.
4. The range of the query displayed is determined by what the user last interacted with e.g. on the first load of the profiler, we use the relative range e.g `last 15m` just as it is now, if a user then switches to an absolute range, it is displayed as `2024-06-11 11:28:23 **to** 2024-06-11 11:43:23`.

This may feel like an extra click to change the range, but it's the same no of clicks as clicking the "Use absolute start time instead" text.

**Comment by yomete** [comment]
Okay that sounds like a good compromise. Do you reckon the menu should only contain the absolute range picker with an "apply range" button to update the query? Also attached a rough mockup. I left the `Use relative start time instead` text in there as a way to dismiss the menu if you're only interested in **typing** in a relative time.

![Image](https://github.com/parca-dev/parca/assets/9016992/6d6c0227-9f0c-4f83-8f0d-923748684e59)

**Comment by brancz** [comment]
I think our previous picker was pretty close to what I'm imagining. Something to remember, at the time we ran queries immediately when we clicked a relative time range or clicked apply. Now we still need to hit search again.

I think the switcher we used to have (see #4024 again) for absolute/relative was really awesome, and I think just hitting that should do the switch we have today with clicking on the text.
